DNN Platform/DotNetNuke.Abstractions/Portals/IStyleColor.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+
+namespace DotNetNuke.Abstractions.Portals
+{
+    /// <summary>
+    /// Represents a CSS color and its components.
+    /// </summary>
+    public interface IStyleColor

```

Comment:
Is it worth thinking about (1) alpha, (2) HSL, (3) LCH?

Reply:
I don't believe so, dnn-elements expects opaque colors and internally may use transparancy on top of those for like disabled stuff, etc. we provide rgb so that consumers can do math on top of a base color. For instance there was css functions to offset a hue or javascript functions to find if either white or black has a better contrast over a certain rgb value. Also we do have color utilities class in dnn-elements that people can use to get those values out of any color: https://github.com/DNNCommunity/dnn-elements/blob/develop/packages/stencil-library/src/utilities/colorInfo.ts so I think we just need to provide the color and let developers use frontend tools if they need converting.